protocols/v2/subprotocols/mining/src/new_mining_job.rs

```diff
@@ -2,7 +2,7 @@
 use alloc::vec::Vec;
 #[cfg(not(feature = "with_serde"))]
 use binary_sv2::binary_codec_sv2;
-use binary_sv2::{Deserialize, Seq0255, Serialize, Sv2Option, B032, B064K, U256};
+use binary_sv2::{Deserialize, Seq0255, Serialize, Sv2Option, B064K, U256};
```

Comment:
If B032 doesn't exist in the spec, maybe `binary_sv2` shouldn't define it?

More rigorously, should it wiped from the codebase?

Reply:
Oh I see, it's still used for `extranonce` and `extranonce_prefix`.

Out of scope for this PR, but is it really worth having a separate `B0_32` datatype instead of just using `B0_255` and then having the spec limit the extranonce size to 31?

It would be less confusing, because `extranonce_size` is U16 which allows for much bigger values. (U8 would have made more sense, but not worth changing now).

Adding an explanation as to why that limit was picked would be good too.